transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpLoginSecurityFilter.java

```diff
@@ -382,7 +397,13 @@ protected boolean login(NextFilter nextFilter,
                     }
                     return false;
                 }
+            } catch (AccessControlLoginException ace) {
```

Comment:
It seems like AccessControlLoginException is public API, otherwise 403 wouldn't be sent.